vocabularies/Common.xml

```diff
@@ -642,6 +642,11 @@ The minimum structure is
         <Annotation Term="Core.Description" String="If specified as true, there's only one value list mapping and its value list consists of a small number of fixed values" />
       </Term>
 
+      <Term Name="ValueListForValidation" Type="Edm.String" Nullable="false" DefaultValue="" AppliesTo="Property Parameter">

```

Comment:
```suggestion
      <Term Name="ValueListForValidation" Type="Edm.String" Nullable="false" AppliesTo="Property Parameter">
```
I would not provide a default value, let's specify the empty string explicitly

Reply:
that's a good point - I will change that. And would you tend to use the pure qualifier or would you prefer the complete annotationpath like @com.sap.vocabularies.v1.Common.ValueListMapping#mySpecialQualifier?